src/content_stream.cpp

```diff
@@ -43,118 +43,164 @@ void ContentReader::close() {
   }
 }
 
-std::string ContentReader::read() {
-  matched_idx = 0;
+// Limit message sizes to 10 MB to prevent DoS issues
+#define MAX_CONTENT_LENGTH (100000000)
```

Comment:
DAP supports custom messages, which may include binary blob style requests. While I'm not aware of anyone doing this, I'm a bit worried that imposing new arbitrary limits [will break someone's workflow](https://xkcd.com/1172/).

At the very least, can we make this configurable build time or runtime?

Reply:
Makes sense, configurable at runtime seems most ideal.